nanshe/io/xtiff.py

```diff
@@ -260,16 +268,24 @@ def get_standard_tiff_data(new_tiff_filename,
 
     new_tiff_description = []
     with tifffile.TiffFile(new_tiff_filename) as new_tiff_file:
-        new_tiff_array = new_tiff_file.asarray(memmap=memmap)
+        if memmap:
+            try:
+                # tifffile >= 0.13.0
+                new_tiff_array = new_tiff_file.asarray(out="memmap")
+            except TypeError:
+                # tifffile < 0.13.0
+                new_tiff_array = new_tiff_file.asarray(memmap=True)
+        else:
+            new_tiff_array = new_tiff_file.asarray()
```

Comment:
Probably should refactor this mess in the future. Going to table it for now though.